### PR TITLE
Added Defaults for SpaceExplorer

### PR DESCRIPTION
**Pull request by broettge**
I updated the src/gui/3DConnexion/3DConnexion.xml file to include support for the SpaceExplorer.
It includes default button mappings that are consistent with other supported models.

### CODE REVIEW

**Comment by marcin86junior** [comment]
Hello broettge!  I have small request - I have "Manipulator - 3d connection". Can you add to right button -> ViewtoFit?
At the moment 2 buttons don't have setup for Manipulator - nothing happens when you press it.
x
Best regards
Marcin Nowak

**Comment by marcin86junior** [comment]
In left button of Inventor I have:

:
In right button I have:


Do you have any suggestion what to add to left button?

**Comment by marcin86junior** [comment]
I'm not super programist:

<ButtonMap DeviceName="Manipulator???">
   <Mapping>
      <Map Description="1"             KeyCode="0"  DownTime="short" Command="Std_ViewFitAll"/>
      <Map Description="2"             KeyCode="1"  DownTime="short" Command="Std_ViewHome"/>
   </Mapping>
</ButtonMap>

**Comment by marcin86junior** [comment]
The name is: SpaceNavigator
z3

**Comment by marcin86junior** [comment]
<ButtonMap DeviceName="SpaceNavigator">
   <Mapping>
      <Map Description="1"             KeyCode="0"  DownTime="short" Command="   Std_ViewHome"/>
      <Map Description="2"             KeyCode="1"  DownTime="short" Command="Std_ViewFitAll"/>
   </Mapping>
</ButtonMap>

**Comment by marcin86junior** [comment]
I tried to fix button locally - but didn't work:
Rotation working fine.

obraz

**Comment by marcin86junior** [comment]
It's working but I needed to: 
1. go to "Adjustments"
2. Setup buttons
3. Chose SpaceNavigator
4. Restart
5. Setup
6. Working - Fit to view :) 2nd button

obraz

**Comment by marcin86junior** [comment]
<ButtonMap DeviceName="3Dconnexion SpaceNavigator">
   <Mapping>
      <Map Description="1"             KeyCode="0"  DownTime="short" Command="Std_ViewHome"/>
      <Map Description="2"             KeyCode="1"  DownTime="short" Command="Std_ViewFitAll"/>
      <Map Description="Bottom"        KeyCode="2"  DownTime="long"  Command="Std_ViewFitAll"/>
      <Map Description="Right"         KeyCode="3"  DownTime="short" Command="Std_ViewFitAll"/>
      <Map Description="Left"          KeyCode="4"  DownTime="long"  Command="Std_ViewFitAll"/>
   </Mapping>
</ButtonMap>

<ButtonMap DeviceName="SpaceNavigator">
   <Mapping>
      <Map Description="1"             KeyCode="0"  DownTime="short" Command="Std_ViewHome"/>
      <Map Description="2"             KeyCode="1"  DownTime="short" Command="Std_ViewFitAll"/>
      <Map Description="Bottom"        KeyCode="2"  DownTime="long"  Command="Std_ViewFitAll"/>
      <Map Description="Right"         KeyCode="3"  DownTime="short" Command="Std_ViewFitAll"/>
      <Map Description="Left"          KeyCode="4"  DownTime="long"  Command="Std_ViewFitAll"/>
   </Mapping>
</ButtonMap>

File: src/Gui/3Dconnexion/3DConnexion.xml [hunk 1]
```diff
@@ -35,6 +35,13 @@
    </Mapping>
 </ButtonMap>
 
+<ButtonMap DeviceName="SpaceNavigator">
+   <Mapping>
+      <Map Description="1"             KeyCode="0"  DownTime="short" Command="Std_ViewHome"/>
+      <Map Description="2"             KeyCode="1"  DownTime="short" Command="Std_ViewFitAll"/>
+   </Mapping>
+</ButtonMap>
+
```
Comment (broettge) [inline src/Gui/3Dconnexion/3DConnexion.xml:44]: Added the SpaceNavigator to the list.

**Comment by marcin86junior** [comment]
Hi Broettge, Thank you adding code. One more thing - the name is "3Dconnexion SpaceNavigator" not "SpaceNavigator" - can you fix the name. That's how windows / freecad recognize old manipulator 3d.  Have nice day!

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5210">pipeline status</a> for feature branch [PR_5210](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5210). Pipeline [419732499](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/419732499) was triggered at [53fc2dc](https://github.com/FreeCAD/FreeCAD/pull/5210/commits/53fc2dcd57d448cf6bb10424574e9c7eb658b5fb). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by luzpaz** [comment]
@broettge please squash and rename commit to `Gui: Add 3Dconnexion SpaceNavigator mouse and mappings`

**Comment by wwmayer** [comment]
Squashed as 632af47a0d

**Pull request closed without merging**